jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/UrlResourceFactoryTest.java

```diff
@@ -308,4 +307,14 @@ private static long fileSize(URL url) throws IOException
             return totalRead;
         }
     }
+
+    private static ByteBuffer toBuffer(Resource resource) throws IOException
+    {
+        try (InputStream is = resource.newInputStream())
+        {
+            byte[] buffer = new byte[Math.toIntExact(resource.length())];
+            is.read(buffer);
+            return ByteBuffer.wrap(buffer);
+        }
```

Comment:
This is only reading part of the `InputStream`.

Reply:
I changed that code to using `BufferUtil` instead.